src/ImageryTab.tsx

```diff
@@ -90,14 +89,63 @@ export function ImageCard(props: {
 
 export default function ImageryTab(props: { namespace: string }) {
   const [enablePreviews, setEnablePreviews] = useState(true);
+  const [observedTopics, setObservedTopics] = useState([] as Array<string>);
+  interface ImageMap {
+    [topic: string]: ImageBitmap;
+  }
+  const [observedImages, setObservedImages] = useState({} as ImageMap);
```

Comment:
```suggestion
  const [observedTopics, setObservedTopics] = useState<Array<string>>([]);
  interface ImageMap {
    [topic: string]: ImageBitmap;
  }
  const [observedImages, setObservedImages] = useState<ImageMap>({});
```
try to avoid typing with the `as` typecast syntax, I shouldn't really have used it like that.